o2a/templates/workflow.tpl

```diff
@@ -12,16 +12,25 @@
   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   See the License for the specific language governing permissions and
   limitations under the License.
- #}
+#}
 
 {% for dependency in dependencies %}
 {{ dependency }}
-{%- endfor %}
+{% endfor %}
 
-PARAMS = {{ params | tojson }}
+JOB_PROPERTIES={
+   {% for key, value in job_properties.items() %}
+       '{{key | python_escape}}': {% if value is none %}None{% else %}'{{ value | python_escape }}'{% endif %},
+   {% endfor %}
+}
+CONFIGURATION_PROPERTIES={
+   {% for key, value in configuration_properties.items() %}
+       '{{key | python_escape}}': {% if value is none %}None{% else %}'{{ value | python_escape }}'{% endif %},
```

Comment:
What do you think about adding None to the filter?

Reply:
List is also be usefull.

Reply:
I'd rather do it in the future but I agree it's a very good idea to simplify the templates a lot!

Reply:
As discussed - we will do it in the future